rules/windows/process_creation/win_nltest_query.yml

```diff
@@ -0,0 +1,24 @@
+title: Nltest Credential Hash Theft
+id: 5cc90652-4cbd-4241-aa3b-4b462fa5a248
+description: Detects nltest query commands which may leak credential hashes
+references:
+    - https://twitter.com/sysopfb/status/986799053668139009
+date: 2018/04/18
+modified: 2020/10/06
+tags:
+    - attack.credential_access
+    - attack.t1003
+status: experimental
+author: Craig Young
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            - nltest
+            - \query
```

Comment:
could you please put `nltest.exe` to the `image` field? that would be cheaper (more efficientent) from resource point of view.

Reply:
Thank you for the pointer. I've committed an update for this.